beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactory.java

```diff
@@ -291,16 +288,7 @@ private SafeFuture<Void> setPayloadOrPayloadHeader(
                         // from the builder bid
```

Comment:
We can create similar method to `getFallbackDataRequired()` like `getBuilderBidRequired()` to make it bit cleaner